src/chapters/booleans-and-conditionals/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Goal Setting and Getting into the Right Mindset
+=======================================================
+
+During this studio, we will ask you to think about your mindset when it comes to facing challenges.
+Do challenges deter you or delight you?
+If challenges are something you delight in, you might have a growth mindset.
+A **growth mindset** can be powered by one word, “yet”.
+The word “yet” allows us to acknowledge that we don’t know something AND that we are capable of learning it.
+For example, with “yet”, the sentence “I don’t understand booleans” becomes “I don’t understand booleans `yet`.”
+
+Growing as developers is something that will take our whole careers.
+No developer knows everything.
+As technology evolves, even the most senior developers have to learn new skills.
+Developing a framework for learning, which includes setting achievable goals, is important for being a developer.
+Technology will change and adapt and we should learn to do the same!
+
+When setting goals, those goals should be SMART. SMART stands for:
+
+**S** pecific
```

Comment:
How about putting these in a table?  This would bring in the row shading as well as allow a second column for defining each term.  (Examples are in the TA tips Google doc).

Reply:
The table did not look great, but I think the pull quote will be the happy medium

Reply:
OK.  I'd like to see how it turns out.